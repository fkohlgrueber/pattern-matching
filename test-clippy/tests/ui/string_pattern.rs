
fn main() {
    let x = "abcdef";
}