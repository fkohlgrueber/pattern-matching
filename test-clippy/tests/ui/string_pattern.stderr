error: This is a match for the string pattern. Well Done too!
  --> $DIR/string_pattern.rs:3:13
   |
LL |     let x = "abcdef";
   |             ^^^^^^^^
   |
   = note: `#[forbid(string_pattern)]` on by default

error: aborting due to previous error

